internal/pkg/cli/job_init.go

```diff
@@ -254,6 +253,26 @@ func (o *initJobOpts) RecommendActions() error {
 	return nil
 }
 
+func (o *initJobOpts) validateDuplicateJob() error {
+	_, err := o.store.GetJob(o.appName, o.name)
+	if err == nil {
+		log.Errorf(`It seems like you are trying to init a job that already exists.
+To recreate the job, please run:
+1. %s. Note that the manifest will remain for reusing purpose.
```

Comment:
How about something like, "Note: The manifest file will not be deleted and will be used in Step 2. If you'd prefer a new default manifest, please manually delete the existing one."